gremlin/values.yaml

```diff
@@ -15,9 +15,11 @@ chaoimage:
 nameOverride: ""
 fullnameOverride: ""
 
-nodeSelector: {}
+daemonSetNodeSelector: {}
+chaoNodeSelector: {}
 
-tolerations: []
+daemonSetTolerations: []
+chaoTolerations: []
```

Comment:
It seems like the values for the chart could be more generally grouped under `chaos` and `daemonset`.

Then you could have a `values.yaml` look more like this:
```
chaos:
  tolerations: {}
  nodeSelector: {}

daemonset:
  tolerations: {}
  nodeSelector: {}
```